workspaces/liferay-learn-workspace/client-extensions/liferay-learn-site-initializer/site-initializer/fragments/group/liferay-learn/fragments/public-sites-navigation/index.js

```diff
@@ -100,9 +100,8 @@ const seeAllResultsLink = fragmentElement.querySelector(
 
 const searchSubmitLink = fragmentElement.querySelector('.search-submit');
 
-const searchSuggestionItem = searchSuggestionItemTemplate.content.querySelector(
-	'a'
-);
+const searchSuggestionItem =
```

Comment:
@luisacmn when I am rerunning the formatters it is reverting these changes. Currently what I do is run `gw formatSource` from the liferay-learn-workspace folder and then run `yarn format` from the workspaces parent folder

Reply:
`yarn format` doesn't work for me. I tried to run it on learn's parent (workspaces)